src/test/java/com/google/javascript/clutz/types_externs_with_platform.d.ts

```diff
@@ -132,3 +132,16 @@ declare namespace ಠ_ಠ.clutz.namespace.subNamespace {
 declare namespace ಠ_ಠ.clutz.namespace.subNamespace {
   var fieldB : number ;
 }
+declare namespace ಠ_ಠ.clutz.window {
```

Comment:
I have seen this behavior too. Under some conditions (externs present), closure starts doubling all global symbols - foo and "window.foo". I am only guessing, but I think es6_min, triggered this behavior. In any case, I replaced this output with a check to not emit anything starting with 'window.'